src/electron/views/device-connect-view/device-connect-view.scss

```diff
@@ -11,5 +11,8 @@
     .main-content-wrapper {
         flex: 1 1 auto;
         overflow: auto;
+        .android-setup-spinner {
```

Comment:
Could this go in a CSS module specific to detect-adb-step?

Reply:
I plan on using it in other android-setup spinners, so what I'll do is create a standalone CSS module in the android-setup folder!